File: week3/0001. Two Sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap = dict()
        for index, num in enumerate(nums):
            anthoer_num = target - num
            if anthoer_num in hashmap:
                return [hashmap[anthoer_num], index]
            hashmap[num] = index
        return None

File: week23/0001. Two Sum/1-two-sum.go
func twoSum(nums []int, target int) []int {
	m := make(map[int]int)
	for i, v := range nums{
		find := target - v
		if _, ok := m[find];ok{
			return []int{m[find], i}
		}
		m[v] = i
	}
	return nil

}